### PR TITLE
PinchDetector components moved to their own transforms, position upda…

### PR DESCRIPTION
**Pull request by protodeep**
…ted with hand, warning if on IHandModel

- [x] verify that pinch demo scenes work properly

- [x] Warning is thrown on Awake() is PinchDetector is directly attached to IHandModel's transform

### CODE REVIEW

File: Assets/LeapMotionModules/PinchUtility/Scripts/LeapPinchDetector.cs [hunk 1]
```diff
@@ -157,6 +160,7 @@ public class LeapPinchDetector : MonoBehaviour {
 
       float pinchDistance = hand.PinchDistance * MM_TO_M;
       transform.rotation = hand.Basis.Rotation();
+      transform.position = hand.PalmPosition.ToVector3();
```
Comment (Amarcolina) [inline Assets/LeapMotionModules/PinchUtility/Scripts/LeapPinchDetector.cs]: This line does nothing, as the position is assigned to a couple lines below.  The transform is already correctly being assigned, so there doesn't need to be anything additional added here.

**Comment by Amarcolina** [comment]
In both scenes the PinchDetector has been moved on the right hand but not the left.

**Comment by Amarcolina** [comment]
Sorry, not both scenes, just the PinchDrawScene

**Comment by protodeep** [comment]
...and done

**Pull request merged**